src/main/scala/devices/debug/Debug.scala

```diff
@@ -135,7 +135,7 @@ case class DebugModuleParams (
   supportHartArray   : Boolean = false,
   hartIdToHartSel : (UInt) => UInt = (x:UInt) => x,
   hartSelToHartId : (UInt) => UInt = (x:UInt) => x,
-  hasImplicitEbreak : Boolean = true
+  hasImplicitEbreak : Boolean = false
```

Comment:
What are the tradeoffs in choosing to use implicit EBREAK? (i.e. why isn't it the default?)